ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -10,6 +10,7 @@
 from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
 from ansible_collections.arista.avd.plugins.filter.range_expand import range_expand
 from ansible_collections.arista.avd.plugins.plugin_utils.avdfacts import AvdFacts
+from ansible_collections.arista.avd.plugins.plugin_utils.strip_empties import strip_null_from_data
 from ansible_collections.arista.avd.plugins.plugin_utils.utils import AristaAvdError, AristaAvdMissingVariableError, default, get, get_item
```

Comment:
This will fail as now the errors should be coming from from `ansible_collections.arista.avd.plugins.plugin_utils.errors`

please rebase so we can verify against the pipeline

Reply:
rebased